folder-management-plugin/grails-app/controllers/org/transmartproject/browse/fm/FmFolderController.groovy

```diff
@@ -230,7 +221,7 @@ class FmFolderController {
     }
 
     def save = {
-        log.info params.toMapString()
+        log.info params
```

Comment:
Won't this result in the following error?

    Caught: groovy.lang.MissingMethodException: No signature of method: ch.qos.logback.classic.Logger.info() is applicable for argument types: (java.util.LinkedHashMap) values: [[:]]
    Possible solutions: info(java.lang.String), info(java.lang.String, [Ljava.lang.Object;), info(java.lang.String, java.lang.Object), info(java.lang.String, java.lang.Throwable), info(org.slf4j.Marker, java.lang.String), info(java.lang.String, java.lang.Object, java.lang.Object)

Reply:
fixed